pkgs/development/python-modules/wandb/default.nix

```diff
@@ -73,6 +73,13 @@ buildPythonPackage rec {
       src = ./hardcode-git-path.patch;
       git = "${lib.getBin git}/bin/git";
     })
+
+    # Avoid `ImportError: cannot import name 'batched' from 'wandb.util'`
```

Comment:
what's the github issue tracking this?

Reply:
There is no issue but PR.
https://github.com/wandb/wandb/pull/6165

I will remove this patch, as a new version was released yesterday.